samples/rag-cosmosdb/python/extensions.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <!-- Locally build and reference the extension -->
+    <ProjectReference Include="../../../src/WebJobs.Extensions.OpenAI.CosmosDBSearch/WebJobs.Extensions.OpenAI.CosmosDBSearch.csproj" />
```

Comment:
are you not able to use just the bundles here?

Reply:
Just using the bundles threw this error: error NU1102: Unable to find package Microsoft.Azure.WebJobs.Extensions.OpenAI with version (>= 0.16.0-alpha)